### DISCUSSION TITLE
Image optimization

### DISCUSSION THREAD

**Discussion by santicros** [ElMassimo/iles#45] [Q&A] [answered]
Is there a way to do image optimization, like Eleventy Image (https://www.11ty.dev/docs/plugins/image/) where we give it an image, and it returns metadata and size+format optimized versions?

**Comment by ElMassimo** [comment, marked as answer]
__UPDATE:__ [`@islands/images`](https://github.com/ElMassimo/iles/tree/main/packages/images) is now the recommended way to optimize images, allowing you to define responsive `srcsets` and different formats, powered by [`vite-plugin-image-presets`](https://github.com/ElMassimo/vite-plugin-image-presets).

---

Planning to provide an `@islands/images` plugin with a similar API to Nuxt Image since it's a common use case.

For now, you can try using one of the Roll-up or Vite plugins, such as imgtools.

**Reply by santicros** [reply]
Awesome, thanks!
